mods/logon-logoff-shutdown-sounds.wh.cpp

```diff
@@ -0,0 +1,582 @@
+// ==WindhawkMod==
+// @id              logon-logoff-shutdown-sounds
+// @name            Logon, Logoff & Shutdown Sounds Restored
+// @description     Restores the logon, logoff and shutdown sounds from earlier versions of Windows
+// @version         1.0.0
+// @author          Toru the Red Fox
+// @github          https://github.com/TorutheRedFox
+// @twitter         https://twitter.com/TorutheRedFox
```

Comment:
Please send me (https://x.com/m417z) a direct message with the following content:

> I attest that I'm the sole owner of both this X account (https://x.com/TorutheRedFox) and the following GitHub account: https://github.com/TorutheRedFox